frontend/src/app/main/ui/render.cljs

```diff
@@ -92,7 +95,9 @@
         text-shapes
         (->> objects
              (filter (fn [[_ shape]] (= :text (:type shape))))
-             (mapv second))]
+             (mapv second))
```

Comment:
no vector need here so `map` is better here. also maybe use `(sequence <transducer> objects)` instead

Reply:
ok